config/cellect.yml.hudson

```diff
@@ -2,4 +2,4 @@ development:
   zk_url: zookeeper:2181
 
 test:
-  zk_url: localhost:2181
+  zk_url: zookeeper:2181
```

Comment:
Do we need a running ZK for testing. This change seems to break test env in travis under jruby as it's looking for the ZK server instead of the stubbed ones we use in cellect testing, https://travis-ci.org/zooniverse/Panoptes/jobs/51899142

https://github.com/zooniverse/Cellect/blob/master/lib/cellect/testing.rb via [spec_helper.rb](https://github.com/zooniverse/Panoptes/blob/a45fd93e1b4668be554dd176666fa6ad0ff056fd/spec/spec_helper.rb#L2)